File: Organizing Data Java/src/tdtu/edu/OrganizingData/ReplaceAnArraywithObject/People.java
package tdtu.edu.OrganizingData.ReplaceAnArraywithObject;

public class People {
	private String name;
	private String phone;
	private String address;

	public People() {
	};

	public People(String name, String phone, String address) {
		super();
		this.name = name;
		this.phone = phone;
		this.address = address;
	}

	@Override
	public String toString() {
		return "People [name=" + name + ", phone=" + phone + ", address=" + address + "]";
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getPhone() {
		return phone;
	}

	public void setPhone(String phone) {
		this.phone = phone;
	}

	public String getAddress() {
		return address;
	}

	public void setAddress(String address) {
		this.address = address;
	}
}

File: Organizing Data C#/DesignPatternGK/ReplaceAnArraywithObject/People.cs
using System;

namespace ReplaceAnArraywithObject
{
    class People
    {
        private string _name;
        private string _phone;
        private string _address;

        public string Name
        {
            get { return _name; }
            set { _name = value; }
        }

        public string Phone
        {
            get { return _phone; }
            set { _phone = value; }
        }

        public string Address
        {
            set { _address = value; }
            get { return _address; }
        }

        public People(string name, string phone, string address)
        {
            _name = name;
            _phone = phone;
            _address = address;
        }

        public People() { }
    }
}
